SwiftyStoreKit/ProductsInfoController.swift

```diff
@@ -44,18 +44,29 @@ class ProductsInfoController: NSObject {
     }
     
     let inAppProductRequestBuilder: InAppProductRequestBuilder
+
+    private var spinLock: OSSpinLock
```

Comment:
I'm under the impression that `OSSpinLock` is deprecated in favor of `os_unfair_lock`. Is there a reason you use this here but not in tests and vice versa?